_posts/2020-05-11-BART-Summary.md

```diff
@@ -0,0 +1,73 @@
+
+**Autoregressive, left to right, LM** (GPT-2)<br />
+**Masked LM**(BERT) replace 15% of the token with [MASK] and predict the corresponding words.<br />
+**Permuted LM** (XLNet) left to right, autoregressive LM training but with the order of the words to predict chosen at random. (worth looking into more, very interesting approach)<br />
+**Multitask Masked LM** (UniLM) combination of right-to-left, left-to-right and Bidirectional. ⅓ of the time using each with shared parameters.<br />
+**Masked Seq2Seq** (MASS) masking a span containing 50% of the tokens and train to  predict the masked tokens.<br />
+        
+### **Results of the first experiment**
+![](https://lh6.googleusercontent.com/YxPMuTJc7EY2rFYIXUBVIYMjV-rloyEj2UmgJ6pbxyyMDCWzdwu4KOgRErOKJcmDe4QfC7LO-2bGE6-_0pCF1lRwJfFbjGvBbuk73oFQ8AgMdHAYDNIwDH8HlEcBI15SQKTUMIUhc_8)
+    
+From the results of this first experiments the authors draw some important conclusions.<br />
+**Token masking is crucial** Only the configurations with token masking or its variations achieve consistently great performance on different tasks.<br />
+**Left-to-right pre-training improves NLG** The Classical Language Model objective despite not doing well in inference or QA tasks achieves SOTA on ELI5(Explain Like I'm 5) a NLG benchmark that with longer sequences values coherent and grammatically correct text.<br />
+**Bidirectional encoders are crucial for QA** Ignoring future context hinders the performance of left-to-right models.<br />
+
+One Interesting note from the authors in this experiment explains how while pre-training techniques and LM objectives are important they don't tell the whole story. They report that their Permuted Language model performs much worse than XLNet due to some of the valuable architectural innovations introduced in XLNet that are not present in BART.
+    
+### **Results of the Large Scale Pre-training experiment**
+After the comparative experiment, the authors trained a bigger model with 12 layers, transformer-based architecture for autoencoding, and similar hyperparameters to RoBERTa. They used both a form of token masking at 30% and sentence permutation as pre-training text-noising techniques and run the model on 160GB of news, books, stories, and web text, again similarly  to RoBERTa
+![](https://lh4.googleusercontent.com/grHkzO3a2zUtdo-_AkH3sGhWmfGQ9_n4bAD0wm78kzVLpuzYFFdSwycnLIevdGUb5jVMJpGdA46LvZN_k0PDrCCObljSvgfcTo6PHevfpa5ZonMXn-C5tEXsUW1V33akbAIINi7whkA)
+
+BART proves especially capable in abstractive summarization talks especially in the **XSum** benchmark that as oppose to the CNN/DailyMail data-set contains very few examples of summaries where phases are present both in the summary and the original text. Besides surpassing the previous best systems in summarization by a considerable margin BART does well also in NLI tasks and QA where it is on par with the SOTA. 
```

Comment:
Typo: "Talks" should probably be "Tasks.
First sentence of this paragraph is a bit long. Try to reformulate